mix.exs

```diff
@@ -31,27 +29,33 @@ defmodule Mjml.MixProject do
   defp deps do
     [
       {:rustler, "~> 0.21"},
-      {:ex_doc, "~> 0.23", only: :dev, runtime: false}
+      {:ex_doc, ">= 0.0.0", only: :dev, runtime: false}
     ]
   end
 
-  defp description() do
-    "NIF bindings for the MJML Rust implementation (mrml)"
-  end
-
-  defp package() do
+  defp package do
     [
+      description: "NIF bindings for the MJML Rust implementation (mrml)",
       maintainers: ["Paul Götze"],
       licenses: ["MIT"],
       files: ~w(lib native .formatter.exs README* LICENSE* mix.exs),
-      links: %{"GitHub" => @github_url}
+      links: %{"GitHub" => @source_url}
     ]
   end
 
-  defp docs() do
+  defp docs do
     [
       main: "readme",
-      extras: ["README.md"]
+      extras: [
+        "CHANGELOG.md": [],
+        "CODE_OF_CONDUCT.md": [title: "Code of Conduct"],
+        "LICENSE.md": [title: "License"],
+        "README.md": [title: "Overview"]
```

Comment:
It looks like files are ordered alphabetically here? Should the `README.md` be the first item in the list, though - in order give a nicer reading flow when coming to the docs pages?

With using the relative link to the `./CONTRIBUTING.md` I also got a warning for the missing referenced file when running `mix docs`: 

```bash
warning: documentation references file "./CONTRIBUTING.md" but it does not exist
  README.md

```

So this list should get an additional: 

```elixir
  # ...
  "CONTRIBUTING.md": [title: "Contributing"],
  # ...
```

Reply:
> It looks like files are ordered alphabetically here? Should the `README.md` be the first item in the list, though - in order to have a nicer reading flow when coming to the docs pages?

Extra files were not added by alphabetical order (didn't even realize that :smile:). 

Normally we set `README.md` as default `:main` page which, by default, expand all the sections in the left navigation bar (see screenshot below). Hence, these `:extras` files were placed in such order so that reader can have a quick view of all the main sections of the documentation in the left navigation bar as shown below:

![image](https://user-images.githubusercontent.com/134518/135759262-19ce453a-f2a5-4eff-8a33-fb9f35ab18a4.png)

> With using the relative link to the ./CONTRIBUTING.md I also got a warning for the missing referenced file when running mix docs

Fixed. Thanks for noticing that.